client/output.rs

```diff
@@ -232,27 +239,40 @@ pub fn print_log(task: &Task) {
         println!("End: {}", end.to_rfc2822());
     }
 
-    if let Some(stdout) = &task.stdout {
-        if !stdout.is_empty() {
-            println!(
-                "{}",
-                style("Std_out: ")
-                    .with(Color::Green)
-                    .attribute(Attribute::Bold)
-            );
-            println!("{}", stdout);
+    if !task_log.stdout.is_empty() {
+        if let Err(err) = print_task_output(&task_log, true) {
+            println!("Error while parsing stdout: {}", err);
```

Comment:
Should these be "parsing" or "printing"?

Reply:
That should be parsing. This only occurs if some kind of error during deserialization occurred.